linux_new/Jenkinsfile

```diff
@@ -0,0 +1,959 @@
+                                //  echo "Distro = ${AlpDistro}"
+                                //  echo "buildArch = ${AlpArch}"
+                                //  echo "FileName = ${AlpFileName}"
+                                //  echo "FilePath = ${AlpFilePath}"
+                                 CheckAndUpload(Target, AlpDistro, AlpArch, '', '', '', '', '' , AlpFileName)
+                        }
+                    }
+                    if (distro == 'linux') {
+                        echo "Linux Upload"
+                        echo "Preparing For Debian Upload"
+                        def DebFileName = ''
+                        def DebTarget = ''
+                        // Creates list like deb.distribution=stretch;deb.distribution=buster;
+                        deb_distros.each { deb_version ->
+                        distro_list += "deb.distribution=${deb_version};"
+                        }
+                        def DebDistro = "Debian"
+                        def DebPackFiles = findFiles(glob: "**/build/ospackage/temurin-*.deb") // List All Packages To Upload
+                        for (DebPackFile in DebPackFiles) {
+                            DebFileName = DebPackFile.name
+                            def DebFilePath = DebPackFile.path
+                            DebTarget = "sfr-deb-test/pool/main/t/temurin-${Release}" // ZZZZZZZZZZ
+                            // echo "Debug 1001 - Debian Upload Debugging"
+                            // echo "Target = ${DebTarget}"
+                            // echo "Distro = ${DebDistro}"
+                            // echo "buildArch = ${arch}"
+                            def DebArchEx = (DebFileName =~ /_(\w+)\.deb$/)
+                            def DebArch = DebArchEx[0][1]
+                            // echo "DebArch = ${DebArch}"
+                            // echo "Release = ${Release}"
+                            // echo "FileName = ${DebFileName}"
+                            // echo "FilePath = ${DebFilePath}"
+                            // echo "DistroList = ${distro_list}"
+                            CheckAndUpload(DebTarget, DebDistro, DebArch,, '', distro_list, '', '', DebFileName, DebPackFile.path)
+                        }
+                        echo "Preapring For RHEL Upload"
+                        def RHELFileName = ''
+                        def RHELFilePath = ''
+                        def RHELTarget = ''
+                        def RHELarchitecture = ''
+                        def RHELDistro = "RPMS"
+                        def RHELPackFiles = findFiles(glob: "**/rhel/build/ospackage/t*.rpm") // List All Packages To Upload For Temurin
+                        for (RHELPackFile in RHELPackFiles) {
+                            RHELFileName = RHELPackFile.name
+                            RHELFilePath = RHELPackFile.path
+                            def RHELarchex = RHELFileName =~ /(?:\d+\.\d+-\d+)\.(\w+)\.(?:src\.)?rpm/
```

Comment:
Perhaps add a comment to say what's happening here with this variable

Reply:
Done